.github/workflows/qns-comment.yml

```diff
@@ -0,0 +1,56 @@
+# Post test results as pull request comment.
+#
+# This is done as a separate workflow as it requires write permissions. The
+# tests itself might run off of a fork, i.e. an untrusted environment and should
+# thus not be granted write permissions.
+
+name: Comment on the pull request
+
+on:
+  workflow_run:
+    workflows: ["QUIC Network Simulator"]
+    types:
+      - completed
```

Comment:
The QNS workflow is currently really slow, since it builds Intel and ARM images, and building the ARM image is done via qemu. It might be worthwhile redoing the QNS workflow so this can kick off once the Intel build has succeeded.

Reply:
:+1: Good point. For now, I skip the ARM build on pull requests (c840abfbff576d649de9a20b5337da288bb1a164). I assume this is a valid shortcut?!